### PR TITLE
ForgotPassword/ResetPassword UI

### PR DESCRIPTION
**Pull request by GabisPapin**
-Created forgot password page - the client can send the email to receive a link;
-created restore password page - the client click on the link in the email and be redirect to a restore password page;
-connected forgot password front and backend;
-styled the both pages.

### CODE REVIEW

**Comment by GabisPapin** [comment]
Forgot-password page
forgot-password-page

**Comment by GabisPapin** [comment]
restore-password page
reset-password-page

File: src/components/forgotPassword/forgotPassword.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React, { useState } from 'react';
+import React from 'react';
+// import { useNavigate } from 'react-router-dom';
```
Comment (lyudad) [inline src/components/forgotPassword/forgotPassword.tsx]: delete comments

File: src/components/forgotPassword/forgotPassword.tsx [hunk 2]
```diff
@@ -16,7 +17,8 @@ const schema = Yup.object({
 
 const forgotPassword = () => {
 	const [forgot] = useForgotPasswordMutation();
-	const [change, setChange] = useState(true);
+	// const navigate = useNavigate();
```
Comment (lyudad) [inline src/components/forgotPassword/forgotPassword.tsx]: delete

**Pull request merged**